code/game/objects/items/weaponry.dm

```diff
@@ -24,7 +24,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 */
 /obj/item/banhammer/attack(mob/M, mob/user)
 	if(user.zone_selected == BODY_ZONE_HEAD)
-		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a banhammer.</span>", "<span class='userdanger'>[user] is stroking your head with a banhammer.</span>", "<span class='hear'>You hear a banhammer stroking a head.</span>")
+		M.visible_message("<span class='danger'>[user] is stroking the head of [M] with a bangammer.</span>", "<span class='userdanger'>[user] is stroking your head with a bangammer.</span>", "<span class='hear'>You hear a bangammer stroking a head.</span>")

```

Comment:
according to the link posted by spacemaniac, the "is"es here were originally "are"s